providers/shared/components/baseline/baseline.ftl

```diff
@@ -6,6 +6,71 @@
     [#return formatSegmentResourceId(SEED_RESOURCE_TYPE)]
 [/#function]
 
+
+[#function getBaselineLinks baselineProfileName baselineComponentNames idsOnly=true  ]
```

Comment:
Think this is AWS specific so really should be in aws tree?

Reply:
It is intended to be a global lookup. Everyone would have baseline and the baselineComponentNames filter can be used for provider specific lookups